### PR TITLE
[documentation] use the latest slate image

### PR DESCRIPTION
**Pull request by pascal-vs**
Update the Slate documentation:
* use the latest docker image in the CI
* remove all files which are already embedded in the docker image to minimize impact in the repository
* fix few output glitches, English, inner links (anchors)

It remains two inner links not yet fixed. Investigations in progress :mag: ...

### CODE REVIEW

**Comment by pascal-vs** [comment]
about
> It remains two inner links not yet fixed. Investigations in progress :mag: ...

one link has been removed, the other one will be fixed later

File: documentation/slate/source/includes/apis.md [hunk 1]
```diff
@@ -54,29 +53,27 @@ This production period cannot exceed one year.
...
+<h2 id="datasets">Datasets</h2>
 
 Very simple endpoint providing the sets of data that are used in the given coverage.
 
 Those datasets (typically from transport authority in GTFS format), each provided by a
 unique [contributor](#contributors) are forming a [coverage](#coverage).
 
 Contributor providing the dataset is also provided in the response.
-Very usefull to know all the datas that form a coverage.
+Very useful to know all the datas that form a coverage.
```
Comment (stifoon) [inline documentation/slate/source/includes/apis.md]: "all the data" and not "all the datas"

**Review by stifoon — APPROVED** [review]

File: documentation/slate/source/includes/apis.md (hunk 1 above)
Comment (pascal-vs) [reply documentation/slate/source/includes/apis.md]: indeed, did not see this one :)

Comment (pascal-vs) [reply documentation/slate/source/includes/apis.md]: thanks

**Review by benoit-bst — APPROVED** [review]

**Pull request merged**